backend-stub/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "azure-function-express": "^2.0.0",
```

Comment:
Do you think we should remove this package?

Reply:
Yeah probably